packages/origin-ui-core/src/components/bundles/SelectedForSale.tsx

```diff
@@ -76,49 +75,62 @@ export const SelectedForSale = (props: IOwnProps) => {
-                                                    <Typography>
-                                                        {province}, {region}
-                                                    </Typography>
-                                                    <Typography>{device.facilityName}</Typography>
-                                                </div>
-                                            }
-                                            secondary={moment(new Date(cert.creationTime)).format(
-                                                'MMM, YYYY'
-                                            )}
-                                        />
+                    {selected.map(
+                        ({
+                            id,
+                            deviceId,
+                            energy: { publicVolume, privateVolume },
+                            creationTime
+                        }) => {
+                            const energy = publicVolume.add(privateVolume);
+                            const { deviceType, province, region, facilityName } = deviceById(
+                                deviceId,
+                                environment,
+                                devices
+                            );
+                            const type = deviceType.split(';')[0];
+                            return (
+                                <ListItem key={id}>
+                                    <ListItemAvatar>
+                                        <Avatar src={energyImageByType(type)}></Avatar>
+                                    </ListItemAvatar>
+                                    <Grid container spacing={3}>
+                                        <Grid item xs={8}>
+                                            <ListItemText
+                                                primary={
+                                                    <div>
+                                                        <Typography>
+                                                            {province}, {region}
+                                                        </Typography>
+                                                        <Typography>{facilityName}</Typography>
+                                                    </div>
+                                                }
+                                                secondary={moment
+                                                    .unix(creationTime)
+                                                    .format('MMM, YYYY')}
+                                            />
+                                        </Grid>
+                                        <Grid item xs={4}>
+                                            <ListItemText
+                                                primary={EnergyFormatter.format(
+                                                    energy.toNumber(),
```

Comment:
Please don't cast `energy` to number. This might cause issues with big numbers. Forward just `energy` to EnergyFormatter.